known-issues.yml

```diff
@@ -0,0 +1,34 @@
+zib-SOAPReport:
+  ignored issues:
+    StructureDefinition.differential.element[6].sliceName:
+      - message: "sd-pg-13: 'Slice names should be camelCased starting with a lowercase letter.'"
```

Comment:
In de profiling guidelines hebben we het volgende opgeschreven:

> Slices follow the convention of FHIR elements in general, using a camelCased name starting with a lowercase letter. If the slice name represents or starts with an abbreviation, the the entire abbreviation should be lowercased (e.g. 'bsn' instead of 'BSN'). 

Waarom wijken we hier dan van af?

Reply:
Geen goede reden, _soapLine_ zag er iets wat gek uit. Was dit uit de profiling guidelines vergeten. Heb het aangepast. Dank!